cli/cli/commands/run/run.go

```diff
@@ -608,6 +624,23 @@ func parseVerbosityFlag(flags *flags.ParsedFlags) (command_args_run.Verbosity, e
 	return verbosity, nil
 }
 
+// Get the image download flag is present, and parse it to a valid ImageDownload value
+func parseImageDownloadFlag(flags *flags.ParsedFlags) (kurtosis_core_rpc_api_bindings.ImageDownloadMode, error) {
+
+	defaultImageMode := kurtosis_core_rpc_api_bindings.ImageDownloadMode_missing
+
+	imageDownloadStr, err := flags.GetString(imageDownloadFlagKey)
+	if err != nil {
+		return defaultImageMode, stacktrace.Propagate(err, "An error occurred getting the image-download using flag key '%s'", imageDownloadFlagKey)
+	}
+	imageDownload, err := command_args_run.ImageDownloadString(imageDownloadStr)
```

Comment:
I propose we remove the `image_download_enumer` from `command_args_run` and translate `imageDownloadStr` straight to `kurtosis_core_rpc_api_bindings.ImageDownloadMode`. 

This is the only place that `image_download_enumer` is used and doesn't seeeem absolutely necessary, so ideally we could also remove the extra representation of the field in CLI so there are less places to keep track of it. Sorry I didn't catch this on my initial review.

Reply:
Not sure if I agree with skip the internal data structure that represents the concept of the flag (the enum) but I see point that is not needed and this pattern is already used by `parseExperimentalFlag` (although `parseVerbosityFlag` which is very similar to image download uses a intermediary enumeration)

Reply:
fair were also a bit inconsistent in how we're representing flags. I interpreted the fact that `command_args_run` only having `verbosity` in it meant that the module was only for representing arguments that stay in the CLI (verbosity doesn't travel to apic/container engine lib). I don't have high conviction about this one so I'll leave it up to you!